pkg/gce-pd-csi-driver/node_test.go

```diff
@@ -517,6 +586,9 @@ func TestNodeStageVolume(t *testing.T) {
 		if tc.expErrCode != codes.OK {
 			t.Fatalf("Expected error: %v, got no error", tc.expErrCode)
 		}
+		if tc.expResize != resizeCalled {
+			t.Fatalf("Test expected resize to be called: %t but was it called: %t", tc.expResize, resizeCalled)
+		}
```

Comment:
Could be more readable with:

```suggestion
		if tc.expResize == true && resizeCalled == false {
			t.Fatalf("Test did not call resize, but it was expected.")
		}
		if tc.expResize == false && resizeCalled == true {
			t.Fatalf("Test called resize, but it was not expected.")
		}
```

However, can we even get a condition where `tc.expResize=false` and `resizeCalled=true`? I think not because `tc.expResize=false` will skip adding command actions for the test and the moment we hit resize code this will panic with `panic: ran out of Command() actions.`

Reply:
I don't think it's wise to have our test cases tailored to the interworkings of the command executor. I'm fine to expand this into the two cases though as you describe.